builder/azure/arm/config_test.go

```diff
@@ -1380,6 +1380,36 @@ func TestConfigShouldRejectVTPMWhenPublishingToAManagedImage(t *testing.T) {
 	}
 }
 
+func TestConfigShouldRejectSkipGeneralizeMWhenPublishingToAManagedImage(t *testing.T) {
```

Comment:
Small typo and a slight rename suggestion
```suggestion
func TestConfigShouldRejectSkipGeneralizeWhenPublishingManagedImage(t *testing.T) {
```